component-interaction/src/app/components/user-details.component/user-details.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="greenBackground" *ngIf="user.dateOfBirth | isAdult">
+    <h4>{{user.name}}</h4>
+    <p>{{user.email}}</p>
+    <p>{{user.dateOfBirth | userAge}}</p>
+    <br>
+</div>
+
+<div class="greyBackground" *ngIf="!(user.dateOfBirth | isAdult)">
```

Comment:
```suggestion
<div [ngClass]="{adult: user.dateOfBirth | userAge | isAdult}">
```

Reply:
another way to do this:
**[class.adult]="user.dateOfBirth | userAge | isAdult"**